README.md

```diff
@@ -65,7 +65,7 @@ We use [Carthage](https://github.com/Carthage/Carthage) for package management d
 ##### If you're building for iOS, tvOS, or watchOS
 
 1. Install Carthage on your Mac using a download from their website or if using Homebrew `brew install carthage`.
-1. We have already created a `Cartfile` that lists the MSAL library for this project on Github. We use the `/dev` branch.
+1. You must create a `Cartfile` that lists the MSAL library for this project on Github. Example: ` github "AzureAD/microsoft-authentication-library-for-objc" "master"`
```

Comment:
Can we add a description for the example that it will use master branch?
i.e./
For example, to use the master branch for MSAL, ..
Probably better worded, though :)